src/components/cubes.jsx

```diff
@@ -39,7 +39,7 @@ export function Cubes({ app, closeOnClickOutside }) {
 
   function openColumnChooser() {
     addOpen.current = true;
-    forceUpdate();
+    forceUpdate(Date.now());
```

Comment:
I think it would be nicer with a `const forceUpdate = useForce();` hook, that did the `Date.now()` magic behind the scenes, so you'd still just do `forceUpdate()`. That way we can keep the logic in one place and change it easily in the future if needed.

Reply:
Added a useForce hook using a bool instead of Date.now().